administrator/components/com_workflow/Helper/WorkflowHelper.php

```diff
@@ -290,6 +290,40 @@ public static function updateAssociationByItemId($itemId, $stateId, $extension =
 	}
 
 	/**
+	 * Updates multiple associations in the workflow_associations table to a given state
+	 *
+	 * @param   array   $itemIds      ids of content
+	 * @param   int     $stateId      id of state
+	 * @param   string  $extension    extension type
+	 *
+	 * @return boolean
+	 *
+	 * @since  __DEPLOY_VERSION__
+	 */
+	public static function updateAssociationOfItemIdList($itemIds, $stateId, $extension = 'com_content')
+	{
+		try
+		{
+			$db    = Factory::getDbo();
+			$query = $db->getQuery(true);
+
+			$query
+				->update($db->qn('#__workflow_associations'))
+				->set($db->qn('state_id') . '=' . (int) $stateId)
+				->where($db->qn('item_id') . ' IN (' . implode(', ', $itemIds) . ')')
```

Comment:
Make sure, we have an array with(!) values in ```$itemIds```

Reply:
done